### DISCUSSION TITLE
SDK for JS

### DISCUSSION THREAD

**Discussion by ali-forooghie** [ory/kratos#1461] [Q&A] [answered]
Hi, I am using Vuejs to develop self-service-ui UI node as a single-page web-application and there is no clear documentation for JS SDK on its npm page (https://www.npmjs.com/package/@ory/kratos-client) or Kratos self-service-ui git-hub page. Please help me with the usage of this SDK and how to use it for manage requests. Thanks a million.

**Comment by vinckr** [comment, marked as answer]
Hey @FlockOfBirdz,
we provide the auto generated SDKs for the community to use. We generally dont use them ourselves. As such we can't provide more comprehensive documentation apart what is coming from the community. 

That being said I want to support you in any way possible to get this going. 
We can collaborate on starting some documentation for this SDK. 
Maybe the [REST API reference](https://www.ory.sh/kratos/docs/next/reference/api/) can also help a bit.
Did `npm install @ory/kratos-client@v0.6.3-alpha.1 --save` run correctly?

**Comment by ali-forooghie** [comment]
Hey @vinckr, Thanks for your response. Actually, I guess your [REST API refrence](https://www.ory.sh/kratos/docs/next/reference/api/) link was broken but now it is working fine. I prefer to use my own requests instead of SDK to communicate with Kratos but I couldn't find any documentation for how to write my own requests. I guess your REST API reference could help about that so thanks a lot.

**Reply by vinckr** [reply]
(Changed the issue to a discussion, since that seemed more fitting. )

Thanks for pointing the broken link out, I think its a redirect bug in docusaurus, will check it out. 

Usually the API reference should work to write some requests, but it can be quite a challenge if you have little experience in it. 
Let me know how it goes!
And if you want to share some of your work afterwards, we will definitely send you some Ory swag as a small thank you 🙂

**Reply by ali-forooghie** [reply]
Hey @vinckr , Thanks for your help and suggestion. *___*

**Reply by vinckr** [reply]
Hey @FlockOfBirdz ,
Just checking in if you had any chance to continue your work.
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton! 🐝
